test/components/settings/settings_handler.spec.jsx

```diff
@@ -23,9 +23,7 @@ describe('SettingsHandler', () => {
       },
       notifications: [],
     });
-    const wrapper = shallow(<Provider store={store} >
-      <SettingsHandler />
-    </Provider>);
+    const wrapper = shallow(<Provider store={store}><SettingsHandler /></Provider>);
```

Comment:
Can we add an exception here? In my opinion, it was much more readable earlier.

Reply:
```jsx
    const wrapper = shallow(
      <Provider store={store}>
        <SettingsHandler />
      </Provider>
    );
```
?

Reply:
I will rebase that commit, so the git log will look clean :smile: